tests/install_upgrade_operators/console_cli_download/conftest.py

```
@@ -66,3 +82,55 @@ def downloaded_and_extracted_virtctl_binary_for_os(request, all_virtctl_urls, tm
+
+@pytest.fixture(scope="class")
+def updated_cluster_ingress_downloads_spec_links(
+    request, admin_client, hco_namespace, ingress_resource, original_virtctl_console_cli_downloads_spec_links
+):
+    ingress_resource_instance = ingress_resource.instance
+    component_routes_cnv = None
+    for component_route in ingress_resource_instance.status.componentRoutes:
+        if component_route.namespace == hco_namespace.name:
+            component_routes_cnv = component_route
+            break
+    assert component_routes_cnv, (
```

Ruff (0.12.2)

Use of `assert` detected

(S101)